src/promptflow/promptflow/_sdk/operations/_flow_operations.py

```diff
@@ -362,6 +361,8 @@ def _build_flow(
         node_variant: str = None,
         update_flow_tools_json: bool = True,
     ):
+        # TODO: confirm if we need to import this
```

Comment:
why this TODO?

Reply:
Such operations should either be encapsulated inside `_submitter` or move to a utils of higher level